README.md

```diff
@@ -30,12 +30,37 @@ If you are already part of the DACCS Network and details about your node have ch
+  - `location` (latitude and longitude of your organization, optional)
+  - `contact` (an email address that can be used by users to contact you if they have questions about your node)
+  - `links` (contains a list of link objects that describe URLs for each node)
+    - each link object must contain a `rel` and `href` key. 
+    - some link objects are required, see the table below for details
+
+See the [node_registry.example.json](doc/node_registry.example.json) file for example values.
+
+#### Links
+
+The `links` key should contain links that describe and provide access to your node. Please see the table below
+for a description of some link values that may be useful.
+
+| `rel`                                              | Meaning                                                                                                                                                         | Required | 
+|----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|
+| `about`                                            | URL to a summary or purpose of the node                                                                                                                         |          |
+| `author`                                           | URL to the maintainer/institution of the node                                                                                                                   |          |
+| `acl`                                              | Endpoint to Access Control List such as the `/magpie` endpoint or another access management method                                                              |          |
+| `collection`                                       | URL to the `/services` endpoint of the node                                                                                                                     | Yes      |
+| `cite-as` <br> `publication`                       | Attribution by researchers to reference the node when using it for publications                                                                                 |          |
+| `copyright` <br> `license` <br> `terms-of-service` | Legal use of the node                                                                                                                                           |          |
+| `describedby`                                      | URL to full documentation and details of the node, its purpose and so on                                                                                        |          | 
+| `edit`                                             | A self-reference to https://github.com/DACCS-Climate/DACCS-node-registry or anywhere that the specific node registry entry to redirect users where to update it |          | 
```

Comment:
I'm not sure I follow the wording of this entry.

Reply:
Yeah, I'll do a little proofread/edit and make the wording a bit more consistent